akka-http2-support/src/main/scala/akka/http/impl/engine/http2/Http2Blueprint.scala

```diff
@@ -9,11 +9,18 @@ import akka.http.scaladsl.model.HttpRequest
 import akka.http.scaladsl.model.HttpResponse
 import akka.stream.scaladsl.BidiFlow
 import akka.stream.scaladsl.Flow
+import akka.stream.scaladsl.Source
 import akka.util.ByteString
 
+/** Represents one direction of an Http2 substream */
+case class Http2SubStream(initialFrame: StreamFrameEvent, frames: Source[StreamFrameEvent, _]) {
+  def streamId: Int = initialFrame.streamId
```

Comment:
good, was about to comment on previous commit that that would be nice to expose like this here